.github/workflows/auto-fill-pr.yml

```
@@ -0,0 +1,165 @@
+              
+              // Try to extract the main summary (before detailed comments)
+              const summaryMatch = coderabbitSummary.match(/## Summary[\s\S]*?(?=##|$)/i);
+              if (summaryMatch) {
+                summarySection = summaryMatch[0];
+              }
+              
+              // Prepare the auto-filled description
+              newDescription = `<!-- Auto-generated by CodeRabbit -->
+# 🤖 AI-Generated Summary
+
+${summarySection}
```

actionlint (1.7.9)

could not parse as YAML: could not find expected ':'

(syntax-check)